q2_feature_classifier/custom.py

```diff
@@ -15,7 +15,11 @@
 
 class LowMemoryMultinomialNB(MultinomialNB):
     def __init__(self, alpha=1.0, fit_prior=True, class_prior=None,
-                 chunk_size=-1):
+                 chunk_size=0):
```

Comment:
This introduces an inconsistency with, for instance, the `length` parameter for `extract_reads`. I like the negative values because they're clearly not a valid `chunk_size` or read `length`.